### ISSUE TITLE
customize the item list header - remove / add header items

### ISSUE THREAD

**Issue by rich-coe** [lwindolf/liferea#795]
add a preference item to customize columns shown in the item list header.
- date/time  and date/time format (#780) 
- headline 
- state icon
- item number
- favorite icon 
- enclosure icon 
- item tag  (#790)
- teaser (wide view)

separate configuration selection for wide view.

**Comment by tukkek** [comment]
I would also like to suggest:

* Feed title (useful when you1re reading "all unread" items or a whole folder instead of a single feed).
* Read/unread indicator (so that it's easier to toggle read/unread without using the context menu).

In general I think this is a great idea, there's a lot of unused screen space for more useful information - even on the "lower common denominator" of modern 720p widescreen desktop resolutions. Customized columns don't sound like they'd be a hassle to implement or maintain either and would be an objective improvement for the application in every way!

Thanks to everyone involved in creating Liferea, I've just migrated to it recently from Inoreader and am very pleased so far!

**Comment by Leiaz** [comment]
@tukkek You can also toggle read/unread with a middle button click.

**Comment by tukkek** [comment]
Thanks for the tip! Have you considered a "tip of the day" feature to inform users of things like these they might have missed? Should I create an issue here for that?

Thanks again for the top-notch application!

**Comment by rich-coe** [comment]
Be careful.  I've used the middle button mouse click, but sometimes, instead of marking the item unread, it executes a 'delete item.'  I think it has something do with the middle button on my mouse being a mouse wheel, and if the wheel moves at all, it deletes the item.
As a work-around, I've been using the right-mouse button context menu.

**Comment by tukkek** [comment]
Aware that I'm repeating myself since last comment, perhaps this should be its own issue @rich-coe .

**Comment by rich-coe** [comment]
@tukkek like "don't use a mouse with a scroll wheel middle button?" kind of issue?

**Comment by tukkek** [comment]
No, like a "New issue" button on the top-right corner kind of issue.

**Comment by ernstki** [comment]
> I would also like to suggest:
> 
>     * Feed title (useful when you1re reading "all unread" items or a whole folder instead of a single feed).
> 
>     * Read/unread indicator (so that it's easier to toggle read/unread without using the context menu).

* author (useful when you're viewing recent changes for a wiki, for example)

**Comment by lwindolf** [comment]
Closing this. There are currently no plans to add additional features.
